Application/EdFi.Ods.AdminApi/Infrastructure/CommonQueryParams.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace EdFi.Ods.AdminApi.Infrastructure
```

Comment:
Use the namespace without braces, sames as Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetVendorsQuery.cs

Example:
`namespace EdFi.Ods.AdminApi.Infrastructure.Database.Queries;`